frontend/src/components/header/navbar/Navbar.tsx

```diff
@@ -0,0 +1,76 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react"
+import AppBar from "@material-ui/core/AppBar"
+import Toolbar from "@material-ui/core/Toolbar"
+import Typography from "@material-ui/core/Typography"
+import { makeStyles } from "@material-ui/core/styles"
+import PersonOutlineIcon from "@material-ui/icons/PersonOutline"
+import { Link } from "react-router-dom"
+import { RouteBuilder } from "../../../routes"
+import { Button } from "../../design/system/button/index"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1
+  },
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+  title: {
+    flexGrow: 1
+  }
+}))
+
+const Navbar: React.FC = () => {
+  const classes = useStyles()
+
+  return (
+    <div className={classes.root}>
+      <AppBar
+        position="static"
+        css={css`
+          background-color: #000000;
+        `}
```

Comment:
```suggestion
        css={{
          backgroundColor: #000000;
        }}
```
Tak też powinno zadziałać. I żeby zachować consistency nawet przy wyobrze kolorów, polecam korzystać z kolorów dostępnych w material-ui

Reply:
Stworzyłam osobny plik z kolorami z material-ui, które będą się powtarzać w całym projekcie